README.md

```diff
@@ -23,3 +30,39 @@ From the source directory, execute each test file:
     cd data-driven-discretization-1d
     python ./pde_superresolution/integrate_test.py
     python ./pde_superresolution/training_test.py
+
+## Training your own model
+
+We used the scripts in the `pde_superresolution/scripts` directly to run
+training. In particular, see `run_training.py`.
+
+Training data was created with `create_training_data.py`, but can also be
+downloaded from Google Cloud Storage:
+
+- https://storage.googleapis.com/data-driven-discretization-public/training-data/burgers.h5
+- https://storage.googleapis.com/data-driven-discretization-public/training-data/kdv.h5
+- https://storage.googleapis.com/data-driven-discretization-public/training-data/ks.h5
+
+We have two notebooks showing how to train and run parts of our model:
+- [Super resolution of Burgers' equation](https://colab.research.google.com/github/google/data-driven-discretization-1d/blob/master/notebooks/burgers-super-resolution.ipynb)
+- [Time integration of Burgers' equation](https://colab.research.google.com/github/google/data-driven-discretization-1d/blob/master/notebooks/time-integration.ipynb)
```

Comment:
I think the time integration notebook is missing

Reply:
Good catch, fixed.